tests/lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/prefer-ember-test-helpers');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const TEST_FILE_NAME = 'some-test.js';
+
+ruleTester.run('prefer-ember-test-helpers', rule, {
+  valid: [
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { blur } from '@ember/test-helpers';
+
+      test('foo', async function(assert) {
+        await blur();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { find } from '@ember/test-helpers';
+
+      test('foo', async function(assert) {
+        await find();
+      });`,
+    },
+
+    {
+      filename: TEST_FILE_NAME,
+      code: `import { focus } from '@ember/test-helpers';
+
+      test('foo', async function(assert) {
+        await focus();
+      });`,
+    },
```

Comment:
Let's add a valid test case to ensure that the native window method (i.e. `focus`) is allowed to be used in non-test files.

Reply:
Good idea!